deploy/helm/sumologic/templates/service.yaml

```diff
@@ -9,6 +9,10 @@ spec:
   selector:
     app: {{ template "sumologic.labels.app" . }}
   ports:
+  - name: zipkin-write
```

Comment:
Don't we want to add this only when traces are enabled?

Reply:
Yeah, I think so 👍